### DISCUSSION TITLE
Cannot enable csv_db, archiving status issues.

### DISCUSSION THREAD

**Discussion by andyfcx** [bellingcat/auto-archiver#149] [Q&A]
Hi, I am using `v0.11.6`.
I need some help for understanding the setting of console DB and config.yml.
I am using gsheet_db and console db from the initialization.

1. I would like to enable csv_db, I edited the config.yml, but there's no new csv created in the assigned directory.

2. Archiving status remains `Archive in progress` in gsheet_db even completed archiving locally, other columns are not updated either.

How can I fix these situation?

**Comment by msramalho** [comment]
### 1.

please paste a trimmed version of your config file and how you are invoking it.

The csv_db should look something like, assuming `local_archive` exists in your local directory:
```yaml
  csv_db:
    csv_file: "./local_archive/db.csv"
```

**Reply by msramalho** [reply]
and you should also have your databases like
```yaml

  databases:
    - console_db
    - csv_db
    - gsheet_db
```

assuming you are using these 3, console_db can be helpful to debug success/failure cases (related to your 2.)
